.gitpod.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gitpod/workspace-full
+                    
+USER gitpod
+
+# Install custom tools, runtime, etc. using apt-get
+# For example, the command below would install "bastet" - a command line tetris clone:
```

Comment:
Shouldn't these comments be removed?

Reply:
It depends.

Reply:
@ret394 was that meant to be a constructive comment? If so, would you care to elaborate, and explain your perspective in a lot more detail?

Reply:
I recently tried but the other side was so pleasant, an average person can't describe it in polite society.

Anyways, what is the problem here- a security bug? A zero day?

Reply:
Yes it is.